backend/backend/settings.py

```diff
@@ -174,6 +179,7 @@
 
 CSRF_TRUSTED_ORIGINS = [
 		'http://localhost:80',
+		'http://localhost:8080',
```

Comment:
Is the backend sending requests to itself?

Reply:
Shit, no, I got the auth issues with the localstorage that made me try all kinds of things and forgot 🗡️